x-pack/docs/en/watcher/actions/pagerduty.asciidoc

```diff
@@ -143,7 +145,7 @@ key:
 
 . Log in to http://pagerduty.com[pagerduty.com] as an account administrator.
 . Go to **Configuration > Services** and select the PagerDuty service. The 
-service must use the API integration.
+service must use the **Events API V2** integration.
```

Comment:
Would it be possible to update the below screenshot as well?

Reply:
Good point - to be honest I wonder if we should just remove both of the pagerduty UI screenshots on this page, given (a) they'll only go out of date again before long, and (b) they don't add that much really, the textual description should be sufficient.

I'll expand the written steps a bit to make sure it's super clear.

Reply:
That sounds good to me as well since the steps look straight forward.

Reply:
Fab, see latest commits (sorry for not doing it all in 1, I'm being lazy and doing this via the web UI given the machine I'm currently on)

Reply:
Thanks! I'll wait a couple of days before approving, just in case the Data Management team chips in their opinion.